src/components/HathorSelect.js

```diff
@@ -63,7 +66,11 @@ const HathorSelect = ({ options, onSelect, value }) => {
 
   return (
     <div className="custom-select-container" ref={selectRef}>
-      <div className="custom-select" onClick={() => setOpen(!open)}>
+      <div
+        className="custom-select"
+        onClick={() => setOpen(!open)}
+        style={{ backgroundColor: `${background ? background : ''}` }}
```

Comment:
issue: The linter is failing with the message below
```
Unnecessary use of conditional expression for default assignment
```

Reply:
corrected conditional. Waiting review.  🙂